style.css

```diff
@@ -0,0 +1,288 @@
+}
+
+.card__pic {
+  z-index: 1;
+  transform: translate(82px, 37px) rotate(39deg);
+  box-shadow: -2px 0 1px rgba(0, 0, 0, .3), 0 4px 4px rgba(0, 0, 0, .25);
+}
+
+.hidden {
+  display: none;
+}
+
+.flip {
+  margin: 0 auto;
+  display: flex;
+  justify-content: space-around;
+  width: 56.5%;
+  min-height: 1024px;
+}
+
+.card-back__wrap {
+  background: url(img/card.png);
+  align-self: center;
+  width: 218px;
+  height: 341px;
+  cursor: pointer;
+  border-radius: 4%;
+  box-shadow: 1px 1px 2px rgba(0, 0, 0, .7);
+  transition: 0.5s;
+}
+
+/* переворот */
+
+.flip-card {
+  background-color: transparent;
+  width: 218px;
+  height: 341px;
+  perspective: 1000px;
+  cursor: pointer;
+}
+
+.flip-card-inner {
+  position: relative;
+  width: 100%;
+  height: 100%;
+  border-radius: 4%;
+  transition: transform .8s;
+  transform-style: preserve-3d;
+}
+
+.flip-card-inner:hover {
+  box-shadow: 10px 10px 15px rgba(0, 0, 0, .5);
+  transform: translate3d(-10px, -30px, -40px) rotate(-3.08deg);
+}
+
+.flip-card.active .flip-card-inner {
+  transform: rotateY(180deg);
+}
+
+.card-back,
+.card-front {
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  -webkit-backface-visibility: hidden; /* Safari */
+  backface-visibility: hidden;
+}
+
+.card-front {
+  background: url(img/gameOver.png);
+  transform: rotateY(180deg);
+}
+
+.card-front_bug {
+  background: url(img/bag.png);
+  transform: rotateY(180deg);
+}
+/* Медиазапросы */
+
+@media (max-width: 768px) {
+  .main {
+    max-width: 85%;
+    min-height: 815px;
+  }
+
+  .level {
+    margin-bottom: 8.8%;
+  }
+
+  .level__choice {
+    font: normal bold 54px/ 60px 'Roboto', sans-serif;
+    padding: 12% 0 4% 2.4%;
+  }
+
+  .level__difficulty {
+    font: normal bold 15px/ 21px 'Roboto', sans-serif;
+  }
+
+  .game input {
+    font: normal bold 46px/ 59px 'Roboto', sans-serif;
+  }
+
+  .begin {
+    display: flex;
+  }
+
+  .begin__button {
+    font: normal bold 28px/ 30px 'Roboto', sans-serif;
+    width: 302px;
+    height: 92px;
+    margin: 14% 0 0 4.5%;
+  }
+
+  .card {
+    display: none;
+  }
+}
+
+@media (max-width: 375px) {
+  .main {
+    min-height: 700px;
+  }
+
+  .level {
+    margin-bottom: 12.8%;
+  }
+
+  .level__choice {
+    font: normal bold 44px/ 55px 'Roboto', sans-serif;
+  }
+
+  .level__difficulty {
+    font: normal bold 12px/ 21px 'Roboto', sans-serif;
+  }
+
+  .game input {
+    font: normal bold 36px/ 59px 'Roboto', sans-serif;
+  }
+
+  .begin__button {
+    font: normal bold 22px/ 30px 'Roboto', sans-serif;
+    width: 252px;
+    height: 72px;
+  }
+}
+
+@media (max-width: 320px) {
+  .main {
+    min-height: 570px;
+  }
+
+  .level {
+    margin-bottom: 7.8%;
+  }
+
+  .level__choice {
+    font: normal bold 36px/ 45px 'Roboto', sans-serif;
+  }
+
+  .level__difficulty {
+    font: normal bold 10px/ 21px 'Roboto', sans-serif;
+  }
+
+  .game input {
+    font: normal bold 32px/ 59px 'Roboto', sans-serif;
+  }
+}
```

Comment:
Codacy found an issue: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/cardBagg/pullRequest?prid=5876445)